src/core/apolloClient/cacheUtils.ts

```diff
@@ -51,6 +52,10 @@ export const logOut = async (client: ApolloClient<object>, resetLocationHistory?
   // Clear store
   await client.cache.reset()
   updateAuthTokenVar()
+
+  // Close the devtools
+  removeItemFromLS(DEVTOOL_STORAGE_KEY)
```

Comment:
Just wondering
We also close the devtool on login here: https://github.com/getlago/lago-front/pull/2200/files#diff-319e2b0e2419e2e0741f6481be2c924f8158491f2add91e412122af742ff0366R44

Won't that be better to keep this LS devtool for having the same context when user logs in back? Or does that conflict with some logic and it could messup somehow?
Seems that key is only used for the open state but maybe it should hold more?
Just thinking as I type, just wanted to make sure there is no redundant logic here